src/database/tables.js

```diff
@@ -3,7 +3,7 @@ const LOGGER = require('@calzoneman/jsli')('database/tables');
 
 export async function initTables() {
     const knex = require('../database').getDB().knex;
-
+    
```

Comment:
Can you remove this change as well?
Just to help the conflict resolution management in the future.

Reply:
Yea i think that was vscode doing what it wants there
```suggestion
```